client/src/components/Navbar/Navbar.jsx

```diff
@@ -1,53 +1,29 @@
 import React from 'react';
+import './Navbar.css';
 import { AppBar, Toolbar, Typography, Button } from '@mui/material';
-import { useTheme } from '@mui/material/styles'; // Import the useTheme hook
-import { makeStyles } from '@mui/styles';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1,
-  },
-  title: {
-    flexGrow: 1,
-  },
-  button: {
-    border: '1px solid transparent',
-    marginLeft: theme.spacing(2),
-  },
-
-  buttonHover: {
-    '&:hover': {
-      backgroundColor: '#8966f1',
-      transition: 'background-color 0.5s, border 0.5s',
-    },
-  },
-}));
+////////// Matrial UI ///////////
 
 const Navbar = () => {
-  const theme = useTheme();
-  const classes = useStyles();
   return (
-    <div className={classes.root}>
-      <AppBar position='static' color='primary'>
+    <div>
+      <AppBar
+        position='static'
+        sx={{ background: '#212529', paddingBottom: '15px' }}
+      >
         <Toolbar>
-          <Typography
-            className={classes.title}
-            variant='h6'
-            style={{ flexGrow: 1 }}
-          >
+          <Typography variant='h6' style={{ flexGrow: 1 }}>
             AcademiaX
           </Typography>
-          <Button
-            color='inherit'
-            className={`${classes.button} ${classes.buttonHover}`}
-            style={{ marginLeft: theme.spacing(2) }}
-          >
+          <Button variant='contained' sx={{ background: '#495057' }}>
             Login
           </Button>
           <Button
-            className={`${classes.button} ${classes.buttonHover}`}
-            color='inherit'
-            style={{ marginLeft: theme.spacing(2) }}
+            variant='contained'
+            sx={{
+              background: '#495057',
+              marginLeft: '15px', // use camelCase for CSS properties
+            }}
           >
```

Comment:
Add more css files we require a proper list on the top of our code and some additional boxes when you hover through the list just a reminder, when you do the pull request again just add those things.